src/services/component-service.ts

```diff
@@ -113,6 +113,7 @@ export class ComponentService {
     return component
   }
 
+  // TODO Move the component validation to the Bundle validator
```

Comment:
The ticket number could be specified here to be clearer

Reply:
ticket number added